### ISSUE TITLE
Laravel 5.6 compatibility 

### ISSUE THREAD

**Issue by stayallive** [sebdesign/laravel-sri#5]
Looked like it worked just fine as usual on an upgrade but not this time... I overlooked something.

You implement `Illuminate\Contracts\Hashing\Hasher` which added the `info` method since 5.6.

There are 2 options I thing:
- Just remove the implements since it's not doing anything really
- Add an empty stub for the info method since it's not used as a hasher

What are your thoughts?

**Comment by baronkoko** [comment]
Anything new on this? 
![image](https://user-images.githubusercontent.com/27996386/44202029-875a4e00-a153-11e8-9911-1e658acedabe.png)

**Comment by stayallive** [comment]
I don;t think there is much active maintenance currently.

So for a quick fix you can implement the following in your composer.json.

```json
{
    "require": {
        "sebdesign/laravel-sri": "dev-feature/laravel56@dev"
    },
    "repositories": [
        {
            "type": "git",
            "url": "https://github.com/stayallive/laravel-sri.git"
        }
    ]
}
```

(I removed ofcourse my other dependencies from the file)

**Comment by sebdesign** [comment]
Hello,

I'm sorry for ignoring this issue. I will look into it today!

**Comment by sebdesign** [comment]
Hi, I've tagged v1.0.6 which fixes the issue for Laravel 5.6.
Let me know if something goes wrong.
Thanks!

**Comment by baronkoko** [comment]
Super! Thanks!
